softwerkskammer/lib/groupsAndMembers/groupsAndMembersService.js

```diff
@@ -102,6 +102,18 @@ module.exports = {
     });
   },
 
+  getOrganizersOfGroup: function (groupId, callback) {
+    this.getGroupAndMembersForList(groupId, (error, group) => {
+      if (error) {return callback(error);}
+      if (group === undefined) {
```

Comment:
Any reason not to use
```suggestion
      if (!group) {
```

Reply:
Thx, still getting used to javascript :)